### DISCUSSION TITLE
Event not fired inside hook

### DISCUSSION THREAD

**Discussion by arcanys-ivan** [feathersjs/feathers#3401] [Q&A]
I have a service that has a hook that also patch/update other service inside of it

i use `await app.service('my-service').patch('id', data)`

but theres no realtime event fired

**Comment by daffl** [comment]
What are you expecting? How does the client/event listener look like? Did you set up the [proper channels](https://feathersjs.com/api/channels.html) (by default a user needs to be authenticated to receive events)?

**Reply by arcanys-ivan** [reply]
this is resolve, I realize im at our microservice, will close this topic now

Thank You
